bin/MadGraph5_aMCatNLO/cards/production/2017/QCD/QCD_HT_LO_MLM/QCD_HT1000to1500/QCD_HT1000to1500_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+import model sm-ckm_no_b_mass
+define vl = ve vm vt
```

Comment:
it seems to me that for these datacards the definition of the neutrinos and leptons is not needed, since only jets are produced

Reply:
Indeed, also the definition of neutrinos and leptons is provided by Madgraph: 
MG5_aMC>display multiparticles
Multiparticle labels:
all = g u c d s u~ c~ d~ s~ a ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ t b t~ b~ z w+ h w- ta- ta+
l- = e- mu-
j = g u c d s u~ c~ d~ s~
vl = ve vm vt
l+ = e+ mu+
p = g u c d s u~ c~ d~ s~
vl~ = ve~ vm~ vt~

So, in any case this lines can be safely removed.